SpinCore_pp/run_field_dep_justMw.py

```diff
@@ -94,7 +94,7 @@ def verifyParams():
 acq_time = nPoints/SW_kHz # ms
 tau_adjust = 0
 deblank = 1.0
-tau = 1000.
+tau = 3500.
```

Comment:
don't commit trivial changes

Reply:
well I thought this was kind of important and we decided to increase tau to 3500 or higher only